erpnext/stock/doctype/stock_reservation_entry/stock_reservation_entry.py

```
@@ -33,7 +33,13 @@ class StockReservationEntry(Document):
 		from_voucher_detail_no: DF.Data | None
 		from_voucher_no: DF.DynamicLink | None
 		from_voucher_type: DF.Literal[
-			"", "Pick List", "Purchase Receipt", "Stock Entry", "Work Order", "Production Plan"
+			"",
```

Ruff (0.13.3)

Syntax error in forward annotation: Expected an expression

(F722)